contracts/lending/NFTFiLending.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: MIT
```

Comment:
The GlobalBNPL contract should use delegatecall for the borrow/repay functions. let's change the implementations to act accordingly